### DISCUSSION TITLE
Tooling: Bundler

### DISCUSSION THREAD

**Discussion by krowter** [ngobrolin/web#22] [Ideas]
Membahas proses bundling (apa dan kenapa perlu), nanti bisa ke source maps, optimasi treeshaking dan minification.

**Comment by rizafahmi** [comment]
Sudah pernah dibahas di episode 39 😎
Apakah sekiranya ada bahasan yang kurang? Kalau ada mungkin bisa kita list disini nanti bisa kita bikin episode bundler kedua. Terimakasih masukannya 🙏 


[![Watch the video](https://i.ytimg.com/vi/bwpPxqgFyWQ/maxresdefault.jpg)](https://youtu.be/bwpPxqgFyWQ?feature=shared)

**Reply by krowter** [reply]
Oh kelewatan berarti saya om. Haha, maaf🙏

**Reply by rizafahmi** [reply]
Santai. ngga perlu minta maaf. Coba ditonton dulu aja, siapa tau ada bahasan yang terlewat yang bisa kita bahas (lagi)
